content/docs/setup/kubernetes/multicluster-install/gateways/index.md

```diff
@@ -0,0 +1,242 @@
+---
+title: With Gateways
```

Comment:
I don't love the titles "With Gateways" and "With VPN" on the page but I do think it looks good in the sidebar nav.  Not sure if there's a way to directly set the sidebar value only.

Reply:
I don't think there's a way to make them different. I think that titles like "Gateway connected clusters" and "VPN connected clusters" might be better.

Reply:
There are many ways to make them different. :wink: My suggestion would be:
Installing an Istio mesh across VPN-based clusters

Reply:
Its too big and overflows the sidebar nav. Makes it hard to read